internal/storage/storageutil/object_attrs.go

```diff
@@ -217,5 +217,6 @@ func ConvertMinObjectToObject(m *gcs.MinObject) *gcs.Object {
 		Updated:         m.Updated,
 		Metadata:        m.Metadata,
 		ContentEncoding: m.ContentEncoding,
+		CRC32C:          m.CRC32C,
```

Comment:
Do we know the reason of why CRC32 attribute is a pointer and not the value itself like Generation ?

Reply:
Not sure. Its defined jacobsa/fuse. Never looked into it.

Reply:
I think you meant it was defined in jacobsa/gcloud and we took the code from there. Not a blocker for this PR but I think we can understand and change in future.